### DISCUSSION TITLE
Getting The Tablet_Location Value From R

### DISCUSSION THREAD

**Discussion by DonkeySaddle** [Open-Systems-Pharmacology/Forum#1406] [Models] [answered]
When I try and get the location of the tablet ("...Application_1|Tablet_Location") by running my model in R I just get an array of zeros. Am I doing something wrong, or is it possible that the location is not accounted for in my model? 

Thanks,

DS

**Comment by Yuri05** [comment, marked as answer]
For each tablet formulation different from "Particle dissolution", PK-Sim allows the option `Use as suspension` to be selected. 

If `Use as Suspension` is activated (which is the **default** setting), in the simulation, the tablet will disintegrate in the stomach and the disintegrated particles will migrate along the gastrointestinal tract compartments. For _Particle Dissolution_ it is the only option per construction.
In this case, **Tablet_location is not used in the model and is indeed always zero**. What is used are the **Suspension fractions of different GI segments** - which you should plot instead.

If `Use as Suspension` is deactivated, the tablet will be treated as non-disintegrating tablet with discrete transition in the different gastrointestinal tract compartments. In this case, Tablet_location is used and is not const zero.

**Comment by DonkeySaddle** [comment]
Thanks @Yuri05 , I think that makes sense. 

So in the case where Use as Suspension is activated that means that the RHS of the lumen compartments (and maybe more?), for example the "...Lumen|LowerJejunum|..." will have at least one line that corresponds to pill dissolution but is simply zero since the "tablet is active" parameter is false?

**Reply by Yuri05** [reply]
Please check these discussions:
* https://github.com/Open-Systems-Pharmacology/Forum/discussions/1186
* https://github.com/Open-Systems-Pharmacology/Forum/issues/488

If after that your question is not answered - let me know.

**Reply by DonkeySaddle** [reply]
@Yuri05  checked both discussions, think everything makes sense. Thanks so much.

DS
